exercises/binary-search-tree/src/test/java/BinarySearchTreeTest.java

```diff
@@ -122,26 +122,41 @@ public void sortsSingleElement() {
     public void sortsCollectionOfTwoIfSecondInsertedIsSmallerThanFirst() {
```

Comment:
The test above should insert `2`, not `4` I think according to the [canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/binary-search-tree/canonical-data.json) 🙂 

I also don't think the `createsComplexTree` test is quite correct according to the canonical data 🙂

Reply:
@FridaTveit : as follow :

{
                    "description": "can sort if second number is smaller than first",
                    "property": "sortedData",
                    "input": {
                        "treeData": ["2", "1"]
                    },
                    "expected": ["1", "2"]
                }

Am I checking at wrong location ?
Will you please reverify ?

Reply:
I think the GitHub comment is just showing up in a weird place - @FridaTveit is right, the test `sortsSingleElement` should not use `4`, instead it should be `2`.

The test `sortsCollectionOfTwoIfSecondInsertedIsSmallerThanFirst` has been correctly updated :slightly_smiling_face: